app/components/EditProfileForm.tsx

```diff
@@ -1,17 +1,58 @@
 'use client'
 import { FormEvent } from 'react';
+import { useRouter } from 'next/navigation';
 
-export default function EditProfileForm() {
+export default function EditProfileForm({userData}) {
 
-  /**
-   * Submits new post data
-   * 
-   * @param e - the form submit event
-  */
-  const onSubmitHandler = (e: FormEvent<HTMLFormElement>): void => {
+  const router = useRouter();
+
+  const firstName = userData['data'][0]['firstName'][0]
+  const lastName = userData['data'][0]['lastName'][0]
+  const username = userData['data'][0]['username'][0]
+  const gender = userData['data'][0]['gender'][0]
+  const bio = userData['data'][0]['bio'][0]
```

Comment:
Running into some issues when I try to open the `/EditProfileForm` page

![image](https://github.com/jackfales/yum/assets/88995035/3f8cc829-de13-419a-94c9-51fb392d05fb)

Reply:
@datderek The error happens because when you create a user in cognito there isn't a user added to the neptune database by default. That needs to be implemented, but for now send a request to the `postUser` API and create a user with the same username as your username in cognito.

Reply:
That makes sense! Works now per Sam's suggestion.